apis/lambda/v1alpha1/custom_types.go

```diff
@@ -43,11 +47,39 @@ type CustomFunctionParameters struct {
 	// +optional
 	RoleSelector *xpv1.Selector `json:"roleSelector,omitempty"`
 
-	CustomFunctionVPCConfigParameters CustomFunctionVPCConfigParameters `json:"vpcConfig,omitempty"`
+	// For network connectivity to AWS resources in a VPC, specify a list of security
+	// groups and subnets in the VPC. When you connect a function to a VPC, it can
+	// only access resources and the internet through that VPC. For more information,
+	// see VPC Settings (https://docs.aws.amazon.com/lambda/latest/dg/configuration-vpc.html).
+	CustomFunctionVPCConfigParameters *CustomFunctionVPCConfigParameters `json:"vpcConfig,omitempty"`
+
+	// The code for the function.
+	// +kubebuilder:validation:Required
+	CustomFunctionCodeParameters *CustomFunctionCodeParameters `json:"code"`
```

Comment:
Should this be a pointer if required?

Reply:
I made it so just to be inline with the rest of the CRD. You feel like it should be value?

Reply:
@muvaf I think typically we make `Required` values, right?

Reply:
> I think typically we make Required values, right?

@negz Yes, we do but ACK always uses pointers, no matter required or not. Because the SDK types are pointers and the generator of the conversion function does not have a logic to account for type differences of the fields, hence all fields on CRDs are also pointers. In this case, the field is not used in generated code but I just made it pointer to be inline with other fields in the same CRD but let me change it to a value type

Reply:
Got it - thanks for the context!